package.json

```diff
@@ -131,7 +160,7 @@
     "@types/node": "18.x",
     "@types/source-map-support": "^0.5.8",
     "@types/uuid": "^9.0.6",
-    "@types/vscode": "^1.96.0",
+    "@types/vscode": "^1.76.0",
```

Comment:
This dependency is tied to the versions of VSCode we target. The previous constraints was too recent and prevented us from building the extension:

```console
$ vsce package --out grafana-vscode.vsix
 ERROR  @types/vscode ^1.96.0 greater than engines.vscode ^1.76.0. Either upgrade engines.vscode or use an older @types/vscode version
```